private/bufpkg/bufcheck/buflint/internal/buflintcheck/buflintcheck.go

```diff
@@ -972,3 +977,48 @@ func checkSyntaxSpecified(add addFunc, file protosource.File) error {
 	}
 	return nil
 }
+
+// CheckValidateConstraintsCheck is a check function.
+var CheckValidateConstraintsCheck = newFilesWithImportsCheckFunc(checkValidateConstraintsCheck)
+
+func checkValidateConstraintsCheck(add addFunc, files []protosource.File) error {
+	m := buflintvalidate.NewValidateModule(files, add)
+	for _, file := range files {
+		for _, message := range file.Messages() {
+			for _, field := range message.Fields() {
+				data, ok, err := getDataByExtension(field, validate.E_Field)
+				if err != nil {
+					return err
+				}
+				if !ok {
+					continue
+				}
+				var constraints validate.FieldConstraints
```

Comment:
`constraints := &validate.FieldConstraints{}`

Reply:
resolved in [182d2e2](https://github.com/bufbuild/buf/pull/2377/commits/182d2e21541d6084e75fd345150fa990cde60d62)